packages/lib-classifier/src/components/Classifier/components/SubjectViewer/components/LightCurveViewer/LightCurveViewer.js

```diff
@@ -149,9 +169,9 @@ class LightCurveViewer extends Component {
     }
   }
   
-  getCurrentTransform () {
+  currentTransform () {
```

Comment:
Naa - functions should be verbs, I think you had it named correctly the first time.

Reply:
😆 LOL, no problem, I'll change this back.

Reply:
Woops, didn't see the comment from @srallen. But functions should be verbs :trollface:

Reply:
Oh, yes. My original comment was about the variable naming, i.e. instead of `t`, name it `currentTransform`, so the whole line should be `const currentTransform = this.getCurrentTransform()`

I don't like abbreviated variable names like this because someone not very familiar with the code might not remember while reading through and might have to go back to the original declaration to try to remember what it is.

Reply:
OH! 'doy, that makes more sense, let me fix that. :picard_face_palm: